source/contents.txt

```diff
@@ -35,7 +34,7 @@ project, this Manual and additional editions of this text.
 
    Introduction </introduction>
    Installation </installation>
-   /mongo
+   MongoDB Shell (mongosh) <https://docs.mongodb.com/mongodb-shell/>
```

Comment:
Minor: Can we use a monospaced font for `mongosh` here to be consistent with how it's displayed in References?

Reply:
The top level entry is a link to a page in another repo. Unfortunately we can't use markup. The link in the References section picks up its formatting from a page within the docs repo so we can modify it there. 

It's a little inconsistent, but rather than remove the formatting from References to match the top-level entry, I'm going to leave refernce/mongosh in monospace since that keeps the copy on the linked page consistent with usage elsewhere in the docs. 

Let me know if we should try to find a different resolution

Reply:
Sounds good.